apps/freelance/src/pages/ProposalsList/index.tsx

```diff
@@ -1,27 +1,43 @@
+import { Spin } from 'antd';
 import { ProposalCard } from '@freelance/components';
-import useProperties from '@hooks/useProperties';
 
-import { mockProjectName, mockProposals } from './constants';
-import { ListContainer, StyledProjectName, Wrapper } from './styles';
+import {
+  ListContainer,
+  SpinContainer,
+  StyledProjectName,
+  Wrapper,
+} from './styles';
+import useProposalsList from './useProposalsList';
 
 const ProposalsList = () => {
-  const { skills } = useProperties();
+  const { data, isLoading, isSuccess } = useProposalsList();
+
+  if (isLoading) {
```

Comment:
better make wrapper, as you on each page need spinner when data is loading and have same if|else in each component quite annoying

Reply:
@lyudad created PageWrapper component with optional props isLoading & isSuccess